src/build/genprod.scala

```diff
@@ -37,15 +37,19 @@ object genprod extends App {
     def i: Int    // arity
 
     def typeArgsString(xs: Seq[String]) = xs.mkString("[", ", ", "]")
+    def typeArgsToTupleSyntacticSugarString(xs: Seq[String]) = xs.mkString("(", ", ", ")")
 
     def to              = (1 to i).toList
     def s               = if (i == 1) "" else "s"
     def className       = name + i
     def classAnnotation = ""
     def fileName        = className + ".scala"
     def targs           = to map ("T" + _)
-    def vdefs           = to map ("v" + _)
-    def xdefs           = to map ("x" + _)
+    def tdefs           = to map ("t" + _)
+    def vdefs           = tdefs
+//    def vdefs           = to map ("v" + _)
+    def xdefs           = tdefs
+//    def xdefs           = to map ("x" + _)
```

Comment:
I would personally prefer to keep `vdefs` and `xdefs` as they were before

Reply:
@lrytz Thanks I will update it soon!

Reply:
@lrytz done.

Reply:
@hepin1989 I'm sorry, we mis-understood each other. I would like to keep the `v1`, `v2`, etc in the `FunctionN.apply` methods, and the `x1`, `x2`, etc in `curried` / `tupled`.

Reply:
For example in https://github.com/scala/scala/pull/7427/files#diff-a08baa1c806de52ca5844014bdfc359a

Reply:
@lrytz I can understand that `x` means `f(x)` ,but the `v` means?

Reply:
The way I understand it
  - `v` means "value", the apply method gets values as parameter
  - `x` is an unknown, so when constructing a lmabda `(x1, x2) => ...` looks nice to me

Reply:
Thank you very much.